src/main/java/org/skriptlang/skript/lang/comparator/Comparators.java

```diff
@@ -44,17 +44,16 @@ private Comparators() {}
 	/**
 	 * A default comparator to compare two objects using {@link Object#equals(Object)}.
 	 */
-	public static final Comparator<Object, Object> EQUALS_COMPARATOR = (o1, o2) -> Relation.get(o1.equals(o2));
-
-	/**
-	 * A Java {@link java.util.Comparator} for comparing two objects using {@link #compare(Object, Object)}.
-	 */
-	public static final java.util.Comparator<Object> JAVA_COMPARATOR = (o1, o2) -> compare(o1, o2).getRelation();
+	private static final ComparatorInfo<Object, Object> EQUALS_COMPARATOR_INFO = new ComparatorInfo<>(
+		Object.class,
+		Object.class,
+		(o1, o2) -> Relation.get(o1.equals(o2))
+	);
 
 	/**
 	 * A List containing information for all registered comparators.
 	 */
-	private static final List<ComparatorInfo<?, ?>> COMPARATORS = Collections.synchronizedList(new ArrayList<>());
+	private static final List<ComparatorInfo<?, ?>> COMPARATORS = Collections.synchronizedList(new ArrayList<>(50));
```

Comment:
If all usages of COMPARATORS are synchronized, it doesn't need a synchronizedList too (not currently the case due to the getter below tho)